packages/components/Atoms/Global/scss/settings/_button.scss

```diff
@@ -19,8 +19,12 @@ $rpl-button-danger-hover-background-color: lighten(rpl-color('danger'), 5%) !def
 $rpl-button-danger-text-color: rpl_color('white') !default;
 $rpl-button-active-text-color: $rpl-button-text-color !default;
 $rpl-button-active-background-color: transparent !default;
-$rpl-button-active-background-image: rpl_gradient('primary_gradient_90') !default;
+$rpl-button-primary-active-background-image: rpl_gradient('primary_gradient_90') !default;
+$rpl-button-primary-active-background-color: transparent !default;
+$rpl-button-primary-active-background-image: $rpl-button-primary-active-background-image !default;
```

Comment:
@rubyllamadora Duplicated `$rpl-button-primary-active-background-image` var? And it's assigned to itself? I think it's typo, isn't it?

Reply:
Updated :P